src/content/app/genome-browser/components/zmenu/ZmenuAppLinks.tsx

```diff
@@ -25,6 +25,7 @@ import { ToggleButton as ToolboxToggleButton } from 'src/shared/components/toolb
 import ViewInApp, { UrlObj } from 'src/shared/components/view-in-app/ViewInApp';
 
 import styles from './Zmenu.scss';
+import useGenomeBrowser from '../../hooks/useGenomeBrowser';
```

Comment:
Don't like to be the import statements pedant, but just noticed this import is in the wrong place.

Reply:
Updated it